Library/PTPusherChannel.m

```diff
@@ -425,12 +431,12 @@ - (void)enumerateObjectsUsingBlock:(void (^)(id obj, BOOL *stop))block
 
 - (PTPusherChannelMember *)me
 {
-  return self[self.myID];
+  return _members[self.myID];
```

Comment:
This shouldn't be necessary as PTPusherChannelMembers implements `objectForKeyedSubscript`